libafl/src/executors/forkserver.rs

```diff
@@ -529,6 +531,8 @@ where
     map: Option<SP::ShMem>,
     phantom: PhantomData<S>,
     map_size: Option<usize>,
+    min_input_size: usize,
+    max_input_size: usize,
```

Comment:
This should be an `Option<>` I think, it's easy to think of targets that don't don't need a max input

Reply:
@domenukk Not sure if that's the best way because then we must error when building in case of shared memory fuzzing (previously it did silent truncation)

Reply:
Not an issue to error on wrong settings I think. It's better than introducing a needless constraint

Reply:
In theory you could probably check this at compile time with types, lol